applications/registration-react/src/pages/catalogs-page/catalogs-page.jsx

```diff
@@ -57,6 +57,17 @@ const renderCatalogs = props => {
               itemsCount={getAPIItemsCount(apis, catalog.id)}
             />
           )}
+          {
+            <Catalog
+              catalogId={`${getConfig().conceptRegistrationApi.host}/${
```

Comment:
noe galt med navn. en url(=host+id) kan ikke navnes catalogId. og senere brukes som url

Også, i tillegg catalogId egentlig inneholder Publisher OrgNr. Så vi kan tenke hvilke komponent bruker hvilke navn.

Je anbefaler å gi url her. de andre knappene skal da få relativ url, mens conceptregistration skal gis absolute url.
```
catalogUrl=`${getConfig().conceptRegistrationApi.host}/${catalog.id}`
og
catalogUrl=`/catalogs/${catalogId}/${type}`
```

og igjen, slik endringene kan du gjør i en ant bransj, som skal merges før dette e.g. "refactor(registration-react): Provide catalog url instead of catalogId to the Catalog component"